queries.sql

```diff
@@ -14,4 +14,37 @@ SELECT * from animals WHERE neutered = TRUE;
 
 SELECT * from animals WHERE name != 'Gabumon';
 
-SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-- First Transaction 
+BEGIN TRANSACTION;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+SELECT * FROM animals; 
+
```

Comment:
Great work @NicolasAndrehh 🤝

- [ ]  You seem to have missed another transaction. Kindly delete all records in the animals' table, then roll back the transaction. Kindly check your project requirements for more clarity.